plugins/plugin-site/src/utils/algolia-queries.js

```diff
@@ -12,6 +12,16 @@ function pluginQueries() {
               currentInstalls
               trend
             }
+            healthScore {
+              value
+              version
+              details {
```

Comment:
I don't think details should be indexed by Algolia, only the data needed to render search results (i.e the resulting score)